whd/dispatcher.py

```diff
@@ -84,11 +87,27 @@ def dispatch_push_event(self, push_event):
 
     def _update_pipeline_definition(self, push_event):
         try:
-            update_repository_pipelines(
-                repo_url=push_event.repository().repository_url(),
-                cfg_set=self.cfg_set,
-                whd_cfg=self.whd_cfg,
-            )
+            try:
+                update_repository_pipelines(
+                    repo_url=push_event.repository().repository_url(),
+                    cfg_set=self.cfg_set,
+                    whd_cfg=self.whd_cfg,
+                )
+            except AttributeError as ae:
```

Comment:
as just discussed:
- check for the missing attribute (`hasattr`)
- define a new exception (e.g. `OutdatedCfgError`)
- raise this specific exception iff the specific attr is amiss
- do the retry only upon this specific exception